.gitignore

```diff
@@ -7,6 +7,7 @@
 .DS_STORE
 .vscode
 .idea
+.env
```

Comment:
I added .env here to set the python path in vscode. Else vscode will complain about faulty imports